src/main/java/com/vaadin/training/bugrap/view/MainViewDesign.java

```diff
@@ -2,25 +2,27 @@
 
 import com.vaadin.annotations.AutoGenerated;
 import com.vaadin.annotations.DesignRoot;
+import com.vaadin.ui.Button;
+import com.vaadin.ui.ComboBox;
 import com.vaadin.ui.Label;
-import com.vaadin.ui.NativeSelect;
-import com.vaadin.ui.VerticalLayout;
+import com.vaadin.ui.VerticalSplitPanel;
 import com.vaadin.ui.declarative.Design;
 
-/** 
+/**
  * !! DO NOT EDIT THIS FILE !!
- * 
+ *
  * This class is generated by Vaadin Designer and will be overwritten.
- * 
+ *
  * Please make a subclass with logic and additional interfaces as needed,
  * e.g class LoginView extends LoginDesign implements View { }
  */
 @DesignRoot
 @AutoGenerated
 @SuppressWarnings("serial")
-public class MainViewDesign extends VerticalLayout {
-	protected NativeSelect<java.lang.String> projectSelect;
-	protected Label userName;
+public class MainViewDesign extends VerticalSplitPanel {
+	protected ComboBox<java.lang.String> projectSelect;
```

Comment:
You can set projectSelect-combobox to use Project-entity as it's generic type which makes dealing with value changes easier.